test/controllers/VatObligationsControllerSpec.scala

```diff
@@ -100,11 +101,13 @@ class VatObligationsControllerSpec extends SpecBase with MockVatObligationsServi
   val testVrn: String = "555555555"
   val badVrn: String = "55"
 
+  val controllerComponents: ControllerComponents = stubControllerComponents()
+
   "The GET VatObligationsController.VatObligations method" when {
 
     "called by an authenticated user" which {
 
-      object TestVatObligationsController extends VatObligationsController(new AuthActionImpl(mockAuth), mockVatObligationsService)
+      object TestVatObligationsController extends VatObligationsController(new AuthActionImpl(mockAuth, controllerComponents), mockVatObligationsService, controllerComponents)
```

Comment:
This line and line 240 are particularly long - so much so that Scalastyle is giving a warning about the length. Could you split them into multiple lines.